app/styles/_data-toolbar.less

```diff
@@ -1,6 +1,13 @@
 /* Data toolbar
    - Includes customizations for label filter input and active filter
 ---------------------------------------------------------------------------- */
+// @screen-md-min - (sidebar-left + left + right margin); 992 - (145 + 30 + 30) = 787
+@data-toolbar-filter-max-width:   (@screen-md-min - (@sidebar-left-width-md + @middle-content-container-padding-lg + @middle-content-container-padding-lg));
+@label-filter-default-input-lg: (@label-filter-default-input-md + 190); // 730px
+@label-filter-default-input-md: (@label-filter-default-input-sm + 140); // 540px
+@label-filter-default-input-sm: (@label-filter-default-input-xs + 160); // 400px
```

Comment:
@sg00dwin I don't think adding arbitrary amounts to the previous width variables here adds much value. I'd rather just set pixel sizes for each.

```less
@label-filter-default-input-xs: 240px;
@label-filter-default-input-sm: 400px;
@label-filter-default-input-md: 540px;
@label-filter-default-input-lg: 730px;
```

Or even inline the values if they're only used once.